crates/kumo-tls-helper/src/lib.rs

```diff
@@ -61,7 +88,7 @@ impl std::hash::Hash for RustlsCacheKey {
 
 lruttl::declare_cache! {
 /// Caches TLS connector information for the RFC5321 SMTP client
-static RUSTLS_CACHE: LruCacheWithTtl<RustlsCacheKey, Arc<ClientConfig>>::new("rfc5321_rustls_config", 32);
+static RUSTLS_CACHE: LruCacheWithTtl<RustlsCacheKey, Arc<ClientConfig>>::new("kumo_tls_helper_rustls_config", 32);
```

Comment:
Renaming this cache is effectively an API change, and we should mention it in the changelog under breaking changes in case someone has a policy that is tuning the cache capacity.

The list of caches is automatically extracted and displayed in https://docs.kumomta.com/reference/kumo/set_lruttl_cache_capacity/ when the docs are built.

I think renaming this makes sense, as its scope is potentially increased as part of moving it.

I'd suggest naming it `rustls_client_config`.

```suggestion
static RUSTLS_CACHE: LruCacheWithTtl<RustlsCacheKey, Arc<ClientConfig>>::new("rustls_client_config", 32);
```